content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -278,13 +274,13 @@ The following example extends the `Comment` and `Tag` models with several scalar
 
 ```prisma
 model Comment {
-|  id      Int    @id @default(autoincrement())
-|  title   String
-|  content String
+  |  id      Int    @id @default(autoincrement())
+  |  title   String
+  |  content String
```

Comment:
This character `|` is similar to the previous comment, used to highlight specific lines.